setup.cfg

```diff
@@ -121,7 +121,7 @@ dev =
     pre-commit
     bandit
     GitPython >=3.1.35  # avoid CVE-2023-41040, CVE-2023-40267 and CVE-2023-40590 (reported by `safety check`)
-    safety
+    safety >=2.3.5
```

Comment:
Is this necessary, as it does not seem to solve the issue of Python 3.11 tests not completing successfully?